kedro/framework/cli/starters.py

```diff
@@ -766,6 +805,8 @@ def _make_cookiecutter_args_and_fetch_template(
         "extra_context": config,
     }
 
+    kedro_version_match_starters = _kedro_and_starters_version_identical()
+
     if checkout:
         cookiecutter_args["checkout"] = checkout
```

Comment:
Is this still needed? I looks like this will be overwritten below in all branches.